Unable to build the codebase with mvnd 1.0.0

### Describe the bug

I'm unable to build this codebase with mvnd 1.0.0. I tried the latest main ed77ee257d5

Motivation: https://quarkus.io/blog/quarkus-3-11-3-released/ says there is compatibility with mvnd 1.0.0

Quarkus codebase builds well when using mvnd 1.0-m8-m39

mvnd 1.0.0 build fails with this error:
```
mvnd clean install -Dquickly -Prelocations  -Dno-test-modules
...
[ERROR] Failed to execute goal de.thetaphi:forbiddenapis:3.4:check (verify-forbidden-apis) on project 
  resteasy-reactive-common-processor: IO problem while reading files with API signatures.:
  /Users/rsvoboda/git/quarkus/independent-projects/resteasy-reactive/common/processor
  /${maven.multiModuleProjectDirectory}/.forbiddenapis/banned-signatures-common.txt
   (No such file or directory) -> [Help 1]
```

### Expected behavior

Compilation of Quarkus main works without any error

### Actual behavior

Compilation of Quarkus main works fails with `[ERROR] Failed to execute goal de.thetaphi:forbiddenapis:3.4:check ...` error

### How to Reproduce?

```
sdk use mvnd 1.0.0
sdk use maven 3.9.8
mvnd clean install -Dquickly -Prelocations  -Dno-test-modules
```

```
mvnd -version
Apache Maven Daemon (mvnd) 1.0.0 darwin-aarch64 native client (0cb7cbcbed6fad010b14dcd05acdc0a59685d88d)
Terminal: org.jline.terminal.impl.PosixSysTerminal with pty org.jline.terminal.impl.jni.osx.OsXNativePty
Apache Maven 3.9.8 (36645f6c9b5079805ea5009217e36f2cffd34256)
Maven home: /Users/rsvoboda/.sdkman/candidates/mvnd/1.0.0/mvn
Java version: 21.0.2, vendor: Eclipse Adoptium, runtime: /Users/rsvoboda/.sdkman/candidates/java/21.0.2-tem
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "14.5", arch: "aarch64", family: "mac"
```

### Output of `uname -a` or `ver`

macOS 14.5 Sonoma

### Output of `java -version`

Java version: 21.0.2, vendor: Eclipse Adoptium, runtime: /Users/rsvoboda/.sdkman/candidates/java/21.0.2-tem

### Quarkus version or git rev

ed77ee257d5

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

mvnd 1.0.0

### Additional information

_No response_

--- a/.mvn/maven.config
+++ b/.mvn/maven.config
@@ -1,0 +1,1 @@
+-Dmaven.multiModuleProjectDirectory=${session.rootDirectory}

--- a/independent-projects/parent/pom.xml
+++ b/independent-projects/parent/pom.xml
@@ -73,7 +73,7 @@
         <maven.compiler.argument.testSource>${maven.compiler.testSource}</maven.compiler.argument.testSource>
 
         <!-- maven-enforcer-plugin -->
-        <maven.min.version>3.8.6</maven.min.version>
+        <maven.min.version>3.9.6</maven.min.version>
         <jdk.min.version>${maven.compiler.argument.source}</jdk.min.version>
         <insecure.repositories>ERROR</insecure.repositories>
 

--- a/integration-tests/gradle/update-dependencies.sh
+++ b/integration-tests/gradle/update-dependencies.sh
@@ -87,7 +87,7 @@ echo ''
 echo 'Sanity check...'
 echo ''
 # sanity check; make sure nothing stupid was added like non-existing deps
-mvn -Dscan=false dependency:resolve validate -Dsilent -q -f "${PRG_PATH}" $*
+${PRG_PATH}/../../mvnw -Dscan=false dependency:resolve validate -Dsilent -q -f "${PRG_PATH}" $*
 
 # CI only: verify that no pom.xml was touched (if changes are found, committer forgot to run script or to add changes)
 if [ "${CI:-}" == true ] && [ $(git status -s -u no '*pom.xml' | wc -l) -ne 0 ]

--- a/tcks/resteasy-reactive/update-dependencies.sh
+++ b/tcks/resteasy-reactive/update-dependencies.sh
@@ -90,7 +90,7 @@ echo ''
 echo 'Sanity check...'
 echo ''
 # sanity check; make sure nothing stupid was added like non-existing deps
-mvn -Dscan=false dependency:resolve validate -Dsilent -q -f "${PRG_PATH}" $*
+${PRG_PATH}/../../mvnw -Dscan=false dependency:resolve validate -Dsilent -q -f "${PRG_PATH}" $*
 
 # CI only: verify that no pom.xml was touched (if changes are found, committer forgot to run script or to add changes)
 if [ "${CI:-}" == true ] && [ $(git status -s -u no '*pom.xml' | wc -l) -ne 0 ]

--- a/update-extension-dependencies.sh
+++ b/update-extension-dependencies.sh
@@ -14,7 +14,7 @@ then
   echo ''
   echo 'Building bom-descriptor-json...'
   echo ''
-  mvn -q -e -Dscan=false clean package -f "${PRG_PATH}/devtools/bom-descriptor-json" -Denforcer.skip $*
+  ./mvnw -q -e -Dscan=false clean package -f "${PRG_PATH}/devtools/bom-descriptor-json" -Denforcer.skip $*
 else
   read -n1 -p 'Build the entire project with relocations? [y/n] ' ANSWER
   echo ''
@@ -23,7 +23,7 @@ else
     echo ''
     echo 'Building entire project with relocations...'
     echo ''
-    mvn -q -e -Dscan=false -Dquickly -Dno-test-modules -Prelocations -T0.8C -f "${PRG_PATH}" $*
+    ./mvnw -q -e -Dscan=false -Dquickly -Dno-test-modules -Prelocations -T0.8C -f "${PRG_PATH}" $*
   else
     echo ''
     echo 'Aborted!'
@@ -112,7 +112,7 @@ echo ''
 echo 'Sanity check...'
 echo ''
 # sanity check; make sure nothing stupid was added like non-existing deps
-mvn -Dscan=false dependency:resolve validate -Dsilent -q -f "${PRG_PATH}" -pl devtools/bom-descriptor-json,docs $*
+./mvnw -Dscan=false dependency:resolve validate -Dsilent -q -f "${PRG_PATH}" -pl devtools/bom-descriptor-json,docs $*
 
 # CI only: verify that no pom.xml was touched (if changes are found, committer forgot to run script or to add changes)
 if [ "${CI:-}" == true ] && [ $(git status -s -u no '*pom.xml' | wc -l) -ne 0 ]

